README.md

```diff
@@ -286,6 +206,5 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ] It would be nice if you fix your MIT link in your readme file because it is broken 😨 Kindly fix this link to make your readme descriptive and professional 👍

NOTE: Kindly see this [documentation](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to add a license file to your project 👍 because right now you don't have any license file 🤔